pkg/generator/steps/layout/tpl-new/py-services/_service_/generated/app/server.go

```diff
@@ -0,0 +1,31 @@
+package app
+
+import (
+	gencontext "github.com/mify-io/mify/pkg/generator/gen-context"
+	"github.com/mify-io/mify/pkg/generator/lib/endpoints"
+)
+
+type serverModel struct {
+	TplHeader        string
+	ServiceName         string
+	ApiEndpointEnv         string
+	ApiEndpoint            string
+	MaintenanceEndpointEnv string
+	MaintenanceEndpoint    string
+}
+
+func newServerModel(ctx *gencontext.GenContext) (serverModel, error) {
+	resolved, err := ctx.EndpointsResolver.ResolveEndpoints(ctx.GetServiceName())
+	if err != nil {
+		return serverModel{}, err
+	}
+
+	return serverModel{
+		TplHeader: ctx.GetWorkspace().TplHeaderPy,
```

Comment:
There is some problem with the formatting here. I suggest enabling auto-formatting.